incubator/rethinkdb-cluster/README.md

```diff
@@ -0,0 +1,131 @@
+
+## StatefulSet Caveats
+* https://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/#limitations
+
+## Acknowledgment of Previous Works
+
+I have heavely barrowed and extended code (peer discovery and probe) from the following project to build this Helm Chart and Docker image: https://github.com/rosskukulinski/kubernetes-rethinkdb-cluster.
+
+## Chart Details
+
+This chart implements a dynamically scalable [RethinkDB Cluster](https://www.rethinkdb.com/docs/sharding-and-replication/) using Kubernetes StatefulSets.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+$ helm install --name my-release incubator/rethinkdb-cluster
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the rethinkdb chart and their default values.
+
+| Parameter                               | Description                                                                        | Default                                             |
+|-----------------------------------------|------------------------------------------------------------------------------------|-----------------------------------------------------|
+| `image.name`                            | Custom RethinkDB image name for auto-joining and probe                             | `codylundquist/helm-rethinkdb-cluster`              |
+| `image.tag`                             | Custom RethinkDB image tag                                                         | `0.1.0`                                             |
+| `image.pullPolicy`                      | Custom RethinkDB image pull policy                                                 | `IfNotPresent`                                      |
+| `cluster.replicas`                      | Number of RethinkDB Cluster replicas                                               | `3`                                                 |
+| `cluster.resources`                     | Resource configuration for each RethinkDB Cluster Pod                              | `{}`                                                |
+| `cluster.podAnnotations`                | Annotations to be added to RethinkDB Cluster Pods                                  | `{}`                                                |
+| `cluster.serviceAnnotations`            | Annotations to be added to RethinkDB Cluster Service                               | `{}`                                                |
+| `cluster.persistentVolume.enabled`      | If `true`, persistent volume claims are created                                    | `true`                                              |
+| `cluster.persistentVolume.storageClass` | Persistent volume storage class                                                    | `volume.alpha.kubernetes.io/storage-class: default` |
```

Comment:
I actually did start using `storageClassName` in the PVC.  I also believe that's k8s 1.6+ as well. So I think I should change the pre-reqs to 1.6+. Thoughts?